components/emoji_picker/emoji_picker.jsx

```diff
@@ -115,7 +115,7 @@ export default class EmojiPicker extends React.Component {
                 name: 'flags',
                 className: 'fa fa-flag-o',
                 id: 'emoji_picker.flags',
-                message: 'Fags',
+                message: 'Flags',
```

Comment:
Noticed a typo unrelated to this story.

Reply:
That's quite a typo. Fortunately that gets overridden by translations and isn't actually displayed anywhere under normal circumstances. :-/

Reply:
Nice catch!